source/_integrations/wake_on_lan.markdown

```diff
@@ -143,7 +143,7 @@ shell_command:
 A switch defined with the `wake_on_lan` platform will render in the UI with both 'on' and 'off' clickable actions. If you don't intend to use the `turn_off` functionality, then using a virtual button & automation will look cleaner and less confusing. It will only have one action.
 
 1. First, define a new helper button. 
-    - Go to **{% my helpers title="Settings > Devices & Services > Helpers" %}** and select the **+ Create helper** button. Choose **Button** and give it a name. A button named "Wake PC" will render like this:
+    - Go to **{% my helpers title="Settings > Devices & services > Helpers" %}** and select the **+ Create helper** button. Choose **Button** and give it a name. A button named "Wake PC" will render like this:
```

Comment:
This change the title so it doesn't match how it is displayed in the fronted:
![image](https://github.com/user-attachments/assets/802fea1d-4c9f-40ff-9675-5ef13c3914b0)

Reply:
I think you might be looking at a translation issue:

![CleanShot 2024-07-16 at 22 11 49@2x](https://github.com/user-attachments/assets/3e9dd59c-cfb0-4c4e-8c3b-84ac44f7db28)